docs/cluster-setup-guide/security/rbac.rst

```diff
@@ -0,0 +1,509 @@
+
+.. code:: bash
+
+   det rbac assign-role -u USER_NAME ROLE_NAME
+   det rbac unassign-role -u USER_NAME ROLE_NAME
+   det rbac assign-role -g GROUP_NAME ROLE_NAME
+   det rbac unassign-role -g GROUP_NAME ROLE_NAME
+
+To assign or unassign a role for a user or a group on a particular workspace, use ``-w
+WORKSPACE_NAME`` switch:
+
+.. code:: bash
+
+   det rbac assign-role -u USER_NAME ROLE_NAME -w WORKSPACE_NAME
+   det rbac unassign-role -u USER_NAME ROLE_NAME -w WORKSPACE_NAME
+   det rbac assign-role -g GROUP_NAME ROLE_NAME -w WORKSPACE_NAME
+   det rbac unassign-role -g GROUP_NAME ROLE_NAME -w WORKSPACE_NAME
+
+WebUI
+-----
+
+To assign or unassign a role for a user or a group globally, first go to user or group management
+UI:
+
+#. Click on your username in the upper left corner.
+#. Click "Settings"
+#. See "User Management" and "Group Management" tabs up at the top.
+
+Then, for users:
+
+#. Click triple-dot icon on the right of the user entry.
+#. Select "Edit" option in the dropdown.
+
+For groups:
+
+#. Click triple-dot icon on the right of the group entry.
+#. Select "Add Roles" option in the dropdown.
+
+To assign or unassign a role for a user or a group on a particular workspace,
+
+#. Go to the workspaces page, select the target workspace.
+#. Click "Members" tab at the top.
+
+To create new role assignments, use "Add Members" button at the top To remove existing role
+assignments, click triple-dot menu and select "Remove" option.
+
+.. _rbac-concepts:
+
+*********************
+ About RBAC Concepts
+*********************
+
+User Groups
+===========
+
+User Groups in Determined are organizational units containing one or more Determined users. User
+Groups currently have no inherent functionality and are not directly useful unless paired with the
+RBAC feature. Users with cluster admin permissions may create groups and add as many users to them
+as needed.
+
+RBAC Permissions and Scopes
+===========================
+
+RBAC allows granting users or user groups a permission to do certain actions, such as various API
+calls, on certain resources, such as experiments. This is achieved using role assignments, which are
+comprised of security principal, role, and scope. Roles, in turn, are comprised of permissions.
+
+Security Principal
+------------------
+
+Security Principal is an entity that is performing an action on a resource. Determined supports
+invidividual users or user groups as a security principal.
+
+Scope
+-----
+
+Scope in Determined refers to where a user may exercise their permitted actions and currently has
+two possible values: global and workspace^specific. A global^level permission is valid anywhere in
+Determined, allowing the user to perform the action on any workspace. A workspace^level permission
+restricts actions so that they are only permissible on the specified workspaces. When using
+workspace^level permissions, the admin must specify which workspace(s) the permission is valid for.
```

Comment:
```suggestion
two possible values: global and workspace-specific. A global-level permission is valid anywhere in
Determined, allowing the user to perform the action on any workspace. A workspace-level permission
restricts actions so that they are only permissible on the specified workspaces. When using
workspace-level permissions, the admin must specify which workspace(s) the permission is valid for.
```